src/containers/AccountConnection.jsx

```diff
@@ -266,7 +285,7 @@ class AccountConnection extends Component {
                   />
                 )}
               </ul>}
-            {!connector.dataType &&
+            {!(connector.dataType && connector.dataType.length) &&
```

Comment:
Ok, I think we could be more explicit then.

```jsx
!(connector.dataType && connector.dataType.length)
```

is not very explicit for me, because as a multi-language developer, I need to check what is `false` and what is `true` when number are used as boolean.

```jsx
function atLeastOne(dataType) {
  dataType && dataType.length > 0
}

// then...

!atLeastOne(connector.dataType) 
```